tpch/Makefile

```diff
@@ -1,24 +0,0 @@
-tbl: dbgen
```

Comment:
why remove this file?

Reply:
Most of its commands are replaced by new bash scripts.

Reply:
For now, the data generation and loading is not a simple command.
I think `Makefile` is a little weak to do this.

Reply:
Let me try it. I haven't tried to pass param in Makefile